okhttp/src/main/java/okhttp3/internal/http2/ErrorCode.kt

```diff
@@ -26,6 +26,12 @@ enum class ErrorCode constructor(val httpCode: Int) {
 
   FLOW_CONTROL_ERROR(3),
 
+  SETTINGS_TIMEOUT(4),
```

Comment:
We could use hexadecimal, eg `0x4`, to be consistent with other error code(`CONNECT_ERROR(0xa)`)?